### PR TITLE
BIOWDL-472: Update TALON to version 5.0.

### PR DESCRIPTION
**Pull request by JasperBoom**
### Checklist
- [x] Pull request details were added to CHANGELOG.md.
- [x] Documentation was updated (if required).
- [x] Workflow `parameter_meta` was added/updated (if required).

### CODE REVIEW

File: sample.wdl [hunk 1]
```diff
@@ -93,6 +87,14 @@ workflow SampleWorkflow {
                 dockerImages = dockerImages
         }
 
+        call talon.LabelReads as executeLabelReadsMinimap2 {
```
Comment (rhpvorderman) [inline sample.wdl]: why not just labelReadsMinimap2?

File: sample.wdl [hunk 2]
```diff
@@ -131,26 +126,38 @@ workflow SampleWorkflow {
                     meanQualityByCycle = false,
                     dockerImages = dockerImages
             }
+
+            call talon.LabelReads as executeLabelReadsTranscriptClean {
```
Comment (rhpvorderman) [inline sample.wdl]: same here, `execute` seems redundant to me.

**Review by rhpvorderman — APPROVED** [review]
Using 'execute' troughout the code base results in very long variable names everywhere. I personally do not think it adds to readability. Especially not if `meta {allowNestedInputs: true}` and `womtool inputs` is used. This will give a lot of `Workflow.execute<subwf>.execute<anothersubwf>.execute<task>.option` type options. Execute is only causing visual clutter here in my opinion.

File: sample.wdl (hunk 2 above)
Comment (rhpvorderman) [reply sample.wdl]: As an aside. `execute` is not the WDL name for performing a task, that is `call`. So if this schema were to make sense, `call` should be used as prefix.

The reason execute looks redundant to me is that *all tasks and workflows* are executed. There is no reason to add this execute statement. It distinguishes nothing.

**Comment by JasperBoom** [comment]
> Using 'execute' troughout the code base results in very long variable names everywhere. I personally do not think it adds to readability. Especially not if `meta {allowNestedInputs: true}` and `womtool inputs` is used. This will give a lot of `Workflow.execute<subwf>.execute<anothersubwf>.execute<task>.option` type options. Execute is only causing visual clutter here in my opinion.

I agree actually. I think I started this when I build the pipeline at first, because some of the naming I used for the tasks were conflicting somehow, making `womtools validate` & `cromwell` fail, but I am not experiencing errors now.

**Comment by rhpvorderman** [comment]
@JasperBoom Looking good. Unfortunately I also just noticed the redundant `output` in front of every output. With the outputs all being in the output section. This seems a bit redundant. But merge at your convenience :wink:.

**Comment by JasperBoom** [comment]
> @JasperBoom Looking good. Unfortunately I also just noticed the redundant `output` in front of every output. With the outputs all being in the output section. This seems a bit redundant. But merge at your convenience wink.

I see what you mean. I will certainly tackle it, but will do it in a separate PR :ok_hand:.

**Review by rhpvorderman — APPROVED** [review]

**Pull request merged**